app/connectors/ApplicationClient.scala

```diff
@@ -207,9 +207,9 @@ trait ApplicationClient {
     }
   }
 
-  def getCurrentSchemeStatus(appId: UniqueIdentifier)(implicit hc: HeaderCarrier): Future[Seq[SchemeEvaluationResult]] = {
+  def getCurrentSchemeStatus(appId: UniqueIdentifier)(implicit hc: HeaderCarrier): Future[Seq[SchemeEvaluationResultWithFailureDetails]] = {
     http.GET(s"${url.host}${url.base}/application/$appId/currentSchemeStatus").map { response =>
-      response.json.as[Seq[SchemeEvaluationResult]]
+      response.json.as[Seq[SchemeEvaluationResultWithFailureDetails]]
```

Comment:
Any reason for renaming? I think the previous name works fine or perhaps `SchemeEvalutionResultDetails`

Reply:
It's not a rename, but rather a new class - i'm enhancing SchemeEvaluationResult but only in the context of this currentSchemeStatus response, other parts of the system use SchemeEvaluationResult for things like PHASE1/2/3 scores and i didn't want to change them